### ISSUE TITLE
Cannot download Osara for reaper

### ISSUE THREAD

**Issue by fatihyuksek1** [jcsteh/osara#874]
1. click on the link https://ci.appveyor.com/api/buildjobs/6nxigp4wdg2b56jd/artifacts/installer/osara_2023.1pre-1110,f5055fd7.exe
2. The link will give you an "ERR_TIMED_OUT" error.
The link should work normally and download the setup file for Osara instead
Using microsoft edge latest version on windows 10 64-bit.

**Comment by ScottChesworth** [comment]
That link worked here in Firefox, Chrome and Edge. Are you sure the browser isn't prompting you whether to proceed with/keep the download?

**Comment by fatihyuksek1** [comment]
No, i'm sure that I don't get any notifications. After some thime, the browser gives err_timed_out. How can I check if the site isn't blocked in my region (Turkey)?

**Comment by fatihyuksek1** [comment]
No, it still doesn’t work. The download page refuses to loadY2 марта 2023 г., в 18:24, RDMurray ***@***.***> написал(а):﻿
I can't find anything to suggest that appveyor is blocked in Turkey. @fatihyuksek1  does it still not work?

**Comment by ScottChesworth** [comment]
We're not using AppVeyor anymore so seems reasonable to assume this is resolved.
